iot-hub/Samples/device/DeviceReconnectionSample/DeviceReconnectionSample.csproj

```diff
@@ -9,7 +9,7 @@
 
   <ItemGroup>
     <PackageReference Include="CommandLineParser" Version="2.8.0" />
-    <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.28.0" />
+    <PackageReference Include="Microsoft.Azure.Devices.Client" Version="1.31.0" />
```

Comment:
what about service sample?

Reply:
I see you are only changing reference for device samples!
Is PnP Device sample already updated?

Reply:
PnP Device and service samples are already updated. I will change the service samples (non-pnp) in a different PR.